.eslintrc

```diff
@@ -78,9 +78,9 @@
     "import/newline-after-import": "warn",
     "@typescript-eslint/restrict-template-expressions": "off",
     "react/prop-types": "off",
+    "@typescript-eslint/no-misused-promises": "off",
```

Comment:
why did you turn off this rule?
this was a question from a previous PR that didn't get an answer

Reply:
It started introducing errors unrelated to any changes I had made, even in master. Whatever it was it appears fixed now